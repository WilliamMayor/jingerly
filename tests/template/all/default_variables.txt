This project template can be found in {{ IN }}
It was rendered in {{ OUT }}