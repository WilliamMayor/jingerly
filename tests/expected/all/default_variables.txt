This project template can be found in tests/template/all
It was rendered in tests/actual/all